toolkit/tools/pkg/imagecustomizerlib/customizeboot.go

```diff
@@ -9,12 +9,20 @@ import (
 	"path/filepath"
 	"regexp"
 
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/imagecustomizerapi"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/imagegen/installutils"
 	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
 	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/safechroot"
 )
 
 var (
 	linuxCommandLineRegex = regexp.MustCompile(`\tlinux .* (\$kernelopts)`)
+
+	linuxCommandLineSELinuxRegex = regexp.MustCompile(`\tlinux .* (security=selinux selinux=1( enforcing=1)?) `)
+	linuxCommandLineInsertRegex  = regexp.MustCompile(`\tlinux \$bootprefix/\$mariner_linux `)
+
+	// Finds the SELinux mode line in the /etc/selinux/config file.
+	selinuxConfigModeRegex = regexp.MustCompile(`(?m)^SELINUX=(\w+)$`)
```

Comment:
Thank you for using `(?m)` and not looping line by line like we did in many other places.:)